cmd/containerd-shim-runhcs-v1/pod.go

```diff
@@ -149,8 +149,12 @@ func createPod(ctx context.Context, events publisher, req *task.CreateTaskReques
 		}
 
 		if lopts != nil {
-			err := parent.SetSecurityPolicy(ctx, lopts.SecurityPolicyEnforcer, lopts.SecurityPolicy)
-			if err != nil {
+			if err := parent.SetConfidentialUVMOptions(
+				ctx,
+				uvm.WithSecurityPolicyEnforcer(lopts.SecurityPolicyEnforcer),
+				uvm.WithSecurityPolicy(lopts.SecurityPolicy),
+				uvm.WithSignedUVMMeasurement(filepath.Join(lopts.BootFilesPath, lopts.SignedUVMMeasurementFile)),
```

Comment:
can the `filepath.Join(` be moved into the `uvm.WithSignedUVMMeasurement` function, so that way we (1) align with how `opts.KernelFile` is exposed; and (2) can have the default `SignedUVMMeasurementFile` be `""`, and differentiate between when a measurement file should be there and isnt, and when a measurement file is unwanted.